dev-cpp/gtest-parallel/gtest-parallel-20220621.ebuild

```diff
@@ -0,0 +1,34 @@
+# Copyright 2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{13..14} )
+inherit python-r1
```

Comment:
Looking at the README.md, this is just a script and not meant to be imported by *other* packages right? (at least the svt-av1 ebuild doesn't do that0. I think python-single-r1 would be more fitting, it'd allow to get rid of the python_foreach_impl logic too.

Reply:
Well, that's handy yes but installing for multiple targets when only one is needed is wasteful for users (may not be a big deal here given it's small, but still).

Reply:
Most users won't have multiple targets enabled, generally with python targets even the larger packages arent that bad with multiple targets enabled. If a user feels strongly about this they can easily set gtest-parallel to just one target per package.

Even so with python compat migrations single target packages are more annoying to deal with as a user.

Reply:
The problem is, I can't really justify making it multi-impl with any facts of the package. It should be single.